package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "description": "Internet Computer message board application",
   "dependencies": {
-    "@dfinity/agent": "^0.21.4",
-    "@dfinity/candid": "^0.21.4",
-    "azle": "^0.20.2",
-    "express": "^4.18.2",
+    "@dfinity/agent": "^1.1.1",
+    "@dfinity/candid": "^1.1.1",
```

Comment:
last time I upgraded these to dependencies to their major versions, it broke the contract. 
have you tested the contract after this upgrade?